velox/docs/functions/spark/aggregate.rst

```diff
@@ -11,10 +11,14 @@ General Aggregate Functions
 
     Returns the bitwise XOR of all non-null input values, or null if none.
 
-.. spark:function:: last(x) -> x
+.. spark:function:: first(x, ignoreNull) -> x
```

Comment:
This documentation doesn't match the code. We should document 2 functions: first(x) and first_ignore_null(x), no?